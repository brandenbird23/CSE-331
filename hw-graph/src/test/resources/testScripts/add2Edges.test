# add2Edge Test File
# tests if an edge can be added to a graph

# Create a graph
CreateGraph graph1

# Create a node
AddNode graph1 node1

# Create an edge
AddEdge graph1 node1 node1 edge1
AddEdge graph1 node1 node1 edge2

# print the nodes in the graph
ListNodes graph1