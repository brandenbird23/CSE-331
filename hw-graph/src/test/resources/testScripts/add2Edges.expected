# add2Edges test file
# tests if an edge can be added to a graph

# Create a graph
created graph graph1

# Create a node
added node node1 to graph1

# Create an edge
added edge edge1 from node1 to node1 in graph1
added edge edge2 from node1 to node1 in graph1

# Print the nodes in the graph
graph1 contains: node1